requirements.txt

```diff
@@ -24,12 +24,12 @@ dealer
 flask-collect
 flake8
 passlib
-twill
+https://github.com/alexandre/twill/tarball/master
```

Comment:
Nice! I will fork to QuokkaProject organization and add you as team member, so the best is to release it to PyPi with an alternative name "twill3"